gematria/proto/annotation.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package gematria;
+
+// Contains annotations to supply additional information to the model, such as
+// cache-miss frequencies, or branching related statistics.
+message AnnotationProto {
+  // A name or label for the annotation.
+  string name = 1;
+  // The annotation value, holding information such as measurements or
+  // statistics like event frequency or rate.
+  double value = 2;
+}
```

Comment:
Nit: Please add an empty line at the end (otherwise, our internal checker will complain).